File: src/NocWMS-Client/src/app/models/inventory-cell.ts
import { ReceiptType } from "./ReceiptType";

export class InventoryCell {
    constructor(inventoryCell?: Partial<InventoryCell>) {
        if (inventoryCell) {
            this.id = inventoryCell.id;
            this.inventoryId = inventoryCell.inventoryId;
            this.warehouseCellId = inventoryCell.warehouseCellId;
            this.cellName = inventoryCell.cellName;
            this.receiptType = inventoryCell.receiptType;
            this.receiptId = inventoryCell.receiptId;
            this.num = inventoryCell.num;
            this.updateTime = inventoryCell.updateTime;
        }
    }
    public id: number;
    public inventoryId: number;
    public warehouseCellId: number;
    public cellName: string;
    public receiptType: ReceiptType;
    public receiptId: string;
    public updateTime: string;
    public num: number;
}

File: src/NocWMS-Server/Server.ApiService/Models/InventoryCell.cs
using System;
using KiraNet.Camellia.Infrastructure.DomainModel.Data;
using Server.ApiService.Common;

namespace Server.ApiService.Models
{
    public class InventoryCell : IEntity<int>
    {
        public int Id { get; set; } // WarehouseCellId
        public int InventoryId { get; set; }
        public int Num { get; set; }
        public DateTime UpdateTime { get; set; }
        public ReceiptType ReceiptType { get; set; }
        public string ReceiptId { get; set; }
        public Inventory Inventory { get; set; }
        public WarehouseCell WarehouseCell { get; set; }
    }
}
